catalog/core/catalog-core-subscriptionstore/subscriptionstore-impl/src/test/java/org/codice/ddf/catalog/subscriptionstore/common/SubscriptionPersistorTest.java

```diff
@@ -166,6 +167,14 @@ public void add(String type, Map<String, Object> properties) throws PersistenceE
         }
 
         @Override
+        public void add(String type, Collection<Map<String, Object>> items)
```

Comment:
These classes should really have used Mockito to mock the `PersistenceStore` instead of hand-coding it ☹️